src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -131,18 +131,22 @@ public GenotypeCalculationArgumentCollection clone() {
     /**
      * If there are more than this number of alternate alleles presented to the genotyper (either through discovery or GENOTYPE_GIVEN ALLELES),
      * then only this many alleles will be used.  Note that genotyping sites with many alternate alleles is both CPU and memory intensive and it
-     * scales exponentially based on the number of alternate alleles.  Unless there is a good reason to change the default value, we highly recommend
+     * scales exponentially based on the number of alternate alleles.
+     *
+     * Unless there is a good reason to change the default value, we highly recommend
      * that you not play around with this parameter.
      *
-     * See also {@link #maxGenotypeCount}.
+     * See also {@link #maxGenotypeCount} and {@link org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBArgumentCollection#MAX_ALTS_LONG_NAME}.
+     * This value can be no smaller than one less than the corresponding GenomicsDB argument.  Sites that exceed the
```

Comment:
Don't you mean "This value can be no **larger** than one less than the corresponding GenomicsDB argument"?